File: web-api-doc/src/main/java/com/truenine/component/webapidoc/autoconfig/AutoConfigEntrance.java
package com.truenine.component.webapidoc.autoconfig;

import com.truenine.component.webapidoc.properties.SwaggerProperties;
import org.springframework.boot.context.properties.EnableConfigurationProperties;
import org.springframework.context.annotation.Import;

@Import({
  OpenApiDocConfig.class,
  //OpenApiConfigController.class
})
@EnableConfigurationProperties(SwaggerProperties.class)
public class AutoConfigEntrance {
}

File: schedule/src/main/kotlin/com/truenine/component/schedule/autoconfig/AutoConfigEntrance.kt
package com.truenine.component.schedule.autoconfig

import com.truenine.component.schedule.properties.XxlJobAutoConfigurationProperties
import com.truenine.component.schedule.properties.XxlJobExecutorAutoConfigurationProperties
import org.springframework.boot.context.properties.EnableConfigurationProperties

@EnableConfigurationProperties(
  value = [
    XxlJobAutoConfigurationProperties::class,
    XxlJobExecutorAutoConfigurationProperties::class
  ]
)
class AutoConfigEntrance
